.github/workflows/main.yml

```diff
@@ -29,7 +29,6 @@ env:
   TOOLCHAIN_PATH: ${{ github.workspace }}/cross
 
   CCACHE: 1
-  DELETE_BUILD: 1
```

Comment:
Since this workflow is not executing the tests, this change is not necessary.

Reply:
Actually the workflow should execute this test to prevent regression. This is the main purpose of this PR.

Reply:
Why testing in \`advanced.yml\` is not enough? So far, we are running no tests in \`main.yml\`...

Reply:
I see, I meant advanced.yml.

Reply:
I originally removed it for consistency between yml files. But it's kept now in the latest changes.